static/js/app.js

```diff
@@ -343,11 +343,39 @@ function setTranslatedStringLinks() {
 
 }
 
-document.addEventListener("DOMContentLoaded", () => {
+/*
+    Use MLS country API to get the client's country code, pass that to a new
+    /premium_subscribe_url endpoint to get the subscription link for the
+    country code, and overwrite the URL that the server determined based only
+    on browser langauge.
+*/
+async function setPremiumSubscribeUrlsByCountry() {
+  const mlsUrl = "https://location.services.mozilla.com/v1/country?key=813238a9-a03c-413d-888b-98615e084a71";
```

Comment:
Just to double-check: it's OK for the key to be public? (I mean, it already is now that it's on GitHub, but still...)

Reply:
Yeah, I asked the same question of the MLS team. The keys are not secret - they're just used to attribute and (if necessary) rate-limit the requests.